cores/arduino/ard_sup/Arduino.h

```diff
@@ -83,4 +82,6 @@ extern "C"
 
 #include "variant.h"
 
+#include "Arduino_defines.h"
```

Comment:
What's the purpose of this move?

Reply:
This was to accommodate the overriding the values in the variant file.  Since that is no longer needed, neither is this.